settings/com.endlessm.settings.gschema.override.in

```diff
@@ -24,6 +24,10 @@ icon-theme='EndlessOS'
 font-name='Lato 12'
 document-font-name='Lato Pro 12'
 
+# Declare the IM module as xim (X Input Module)
+[org.gnome.desktop.interface]
+gtk-im-module=xim
```

Comment:
May I ask what's the rationale behind this, and what is this trying to solve? I thought XIM was an obsolete input method. Why not ibus?